### DISCUSSION TITLE
I miss data from my heat pump

### DISCUSSION THREAD

**Discussion by HSoft99** [john30/ebusd#1399] [Q&A]
I use ebusd with my Vaillant heat pump and get some information to mqtt.

![grafik](https://github.com/user-attachments/assets/3d1be84a-10cd-4307-90f0-4bff52c7e9b2)

I miss some information like consumption, ...

**The result of ebusctl info:**
version: ebusd 24.1.24.1
update check: OK, device firmware v5-c6 1[4a1b] up to date
device: IP-Adress:9999, TCP, enhanced, firmware 1.1[4a1b].1[4a1b]
signal: acquired
symbol rate: 43
max symbol rate: 210
scan: finished
reconnects: 0
masters: 5
messages: 84
conditional: 0
poll: 0
update: 10
address 03: master https://github.com/john30/ebusd-configuration/pull/11
address 08: slave https://github.com/john30/ebusd-configuration/pull/11, scanned "MF=Vaillant;ID=HMU00;SW=0902;HW=5103", loaded "vaillant/08.hmu.csv"
address 10: master https://github.com/john30/ebusd-configuration/pull/2
address 15: slave https://github.com/john30/ebusd-configuration/pull/2, scanned "MF=Vaillant;ID=CTLV3;SW=0709;HW=3704"
address 26: slave, scanned "MF=Vaillant;ID=VR_71;SW=0201;HW=0503", loaded "vaillant/26.vr_71.csv"
address 31: master https://github.com/john30/ebusd-configuration/issues/8, ebusd
address 36: slave https://github.com/john30/ebusd-configuration/issues/8, ebusd
address 71: master https://github.com/john30/ebusd-configuration/issues/9
address 76: slave https://github.com/john30/ebusd-configuration/issues/9, scanned "MF=Vaillant;ID=VWZIO;SW=0902;HW=5103"
address f1: master https://github.com/john30/ebusd-configuration/issues/10
address f6: slave https://github.com/john30/ebusd-configuration/issues/10, scanned "MF=Vaillant;ID=NETX3;SW=0121;HW=0404"

**ebusctl scan result**
08;Vaillant;HMU00;0902;5103;21;23;32;0010021115;1610;007452;N0
15;Vaillant;CTLV3;0709;3704;21;23;37;0010045478;0953;050226;N9
26;Vaillant;VR_71;0201;0503;21;24;20;0020184846;0082;009459;N3
76;Vaillant;VWZIO;0902;5103;21;23;25;0010023523;3110;009382;N9
f6;Vaillant;NETX3;0121;0404;21;24;11;8000015501;0933;033133;N5

ebusctl find shows a lot of element with "no data stored". For example ConsumptionTotal, YieldTotal, Broadcast Datetime and more elements

The vaillant app shows me the information. What is the reason, that i don't get the information for my heat pump over ebusd

**Comment by dietersiebs** [comment]
It seems, you have the same components like me.

It seems, you will read your configuration from John, At the beginning, I got only some informations, too. Now I'm working with local csv files, which I edited in some details. Now I get many data points, but I still not see some entries, in the log I see "invalid position".

Where do you want to see your data? Home Assistant?

**Reply by HSoft99** [reply]
Hi dietersiebs,
that's correct, I use the configuration from John and write the data points in a mqtt broker and from there to Openhab. I get the messages from ebusctl and docker log-file for john30/ebusd. 

Can you provide me with your csv files please ?

**Comment by dietersiebs** [comment]
@HSoft99 : I invited you to my private repository, so you can download my actual files. They are not perfect, but almost all enties are returning data.

**Reply by HSoft99** [reply]
Thanks @dietersiebs, I downloaded the files. I won't get around to testing the configuration with my system until next week at the earliest

**Reply by HSoft99** [reply]
Hi @dietersiebs, it works fine. I get a lot of information and need to find out what everything means. Thank you very much

**Comment by dietersiebs** [comment]
By the way:

only entries which are matching the MQTT filter will be send via MQTT. In the ebusd Addon in Home Assistant I had to edit the filter in thie configuration of mqtthassio.cfg. If a name of an entry in the csv-files doesn't match, I get no entity in Home Assistant. My last filter is

"filter-name=z1|name|stat|temp|heat|cool|water|humidity|yield|count|energy|power|consum|point|runtime|hours|starts|mode|curve|^load$|^party$|sensor|time|system|total|day|month|year|manag|charge|scheme||multi|install|number|main|set|mix|zone|name|solar|max|min"

Now I see much more entities than before...

Maybe you habe to edit your configuration similar, too.
